home.css

```diff
@@ -123,3 +124,59 @@ pre {
   overflow-y: auto;
   color: white;
 }
+
+/* .navbar  */
+
+.navbar-container {
+  width: 80%;
+}
+
+.navbar {
+  padding: 0 8.3%;
+  height: 80px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
You can put all the properties that are repeating in one utility class and then use them.
This will make your code cleaner.